StructureDefinition/NHSDigital-Claim.StructureDefinition.xml

```diff
@@ -156,7 +141,7 @@
+      <definition value="The organisation responsible for the claim, this must be a reference to a contained PractitoinerRole&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA; &quot;provider&quot;: {&#xD;&#xA;        &quot;reference&quot; : &quot;#provider&quot;&#xD;&#xA;    },&#xD;&#xA;```&#xD;&#xA;&#xD;&#xA;The contained PractitionerRole&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;{&#xD;&#xA;    &quot;resourceType&quot;: &quot;Claim&quot;,&#xD;&#xA;    &quot;id&quot;: &quot;258E153F-567B-4D60-B045-BA315A6B8DB0&quot;,&#xD;&#xA;    &quot;contained&quot;: [&#xD;&#xA;        {&#xD;&#xA;            &quot;resourceType&quot;: &quot;PractitionerRole&quot;,&#xD;&#xA;            &quot;id&quot;: &quot;provider&quot;,&#xD;&#xA;            &quot;practitioner&quot;: {&#xD;&#xA;                &quot;identifier&quot;: {&#xD;&#xA;                    &quot;system&quot;: &quot;https://fhir.hl7.org.uk/Id/gphc-number&quot;,&#xD;&#xA;                    &quot;value&quot;: &quot;2344242&quot;&#xD;&#xA;                },&#xD;&#xA;                &quot;display&quot;: &quot;|Bob||Buller|&quot;&#xD;&#xA;            },&#xD;&#xA;            &quot;organization&quot;: {&#xD;&#xA;                &quot;identifier&quot;: {&#xD;&#xA;                    &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/ods-organization-code&quot;,&#xD;&#xA;                    &quot;value&quot;: &quot;FA976&quot;&#xD;&#xA;                },&#xD;&#xA;                &quot;display&quot;: &quot;Bob's Pharmacy&quot;&#xD;&#xA;            },&#xD;&#xA;            &quot;telecom&quot;: [&#xD;&#xA;                {&#xD;&#xA;                    &quot;system&quot;: &quot;phone&quot;,&#xD;&#xA;                    &quot;use&quot;: &quot;work&quot;,&#xD;&#xA;                    &quot;value&quot;: &quot;01234567890&quot;&#xD;&#xA;                }&#xD;&#xA;            ]&#xD;&#xA;        }&#xD;&#xA;    ],&#xD;&#xA;```" />
```

Comment:
I can't tell in the GitHub diff whether this renders correctly but I'll assume it does